site/admin/theatres/create.php

```diff
@@ -0,0 +1,107 @@
+<?php
+include_once('init.php');
+PrintHeader('Create Theatre', 'Create Theatre');
```

Comment:
I forgot to mention this in the form training, but you don't want to echo anything until *after* you've done validation. The reason for this is that once you've echo html to the browser, you can't correctly redirect to a different page (it works sometimes, but sometimes it won't work). The redirect happens in the page headers which technically have to come before the actual html content.